lang/main.json

```diff
@@ -1106,42 +1116,48 @@
             "lobbyButton": "Enable/disable lobby mode",
             "localRecording": "Toggle local recording controls",
             "lockRoom": "Toggle meeting password",
+            "lowerHand": "Lower your hand",
             "moreActions": "More actions",
             "moreActionsMenu": "More actions menu",
             "moreOptions": "Show more options",
-            "mute": "Mute / Unmute",
+            "mute": "Mute",
             "muteEveryone": "Mute everyone",
             "muteEveryoneElse": "Mute everyone else",
             "muteEveryoneElsesVideoStream": "Stop everyone else's video",
             "muteEveryonesVideoStream": "Stop everyone's video",
             "noiseSuppression": "Noise suppression",
-            "participants": "Participants",
+            "openChat": "Open chat",
+            "participants": "Open participants pane",
             "pip": "Toggle Picture-in-Picture mode",
             "privateMessage": "Send private message",
             "profile": "Edit your profile",
-            "raiseHand": "Raise / Lower your hand",
-            "reactionsMenu": "Open / Close reactions menu",
+            "raiseHand": "Raise your hand",
```

Comment:
Same question for fullScreen translation.

Reply:
Yeah I noticed that too when editing.

I don't know if there was a good reason for the differences on those cases. You'll know better than me I think :sweat_smile:

Only thing I did was updating the current already used translations looking like "show/hide …", separating them in two "show" + "hide" translations.

I also saw there were lots of translations in `toolbar.accessibilityLabel` that were the exact same as the ones in `toolbar` but since that was the way of doing things already I followed it.

Reply:
Ok I will check those translations and delete them after we merge this.